package.json

```diff
@@ -30,9 +30,6 @@
     "test:storybook": "pnpm turbo build:deps test:storybook",
     "test:storybook:update": "pnpm turbo build:deps test:storybook:update"
   },
-  "dependencies": {
-    "@cartridge/presets": "github:cartridge-gg/presets#c7e6fb7"
```

Comment:
we need to make sure this change works with the presets update ci

Reply:
Good point !